java/src/jmri/jmrit/withrottle/AbstractController.java

```diff
@@ -51,12 +51,12 @@
      * @param manager which manager to get system names for.
      *
      */
-    @SuppressWarnings({"unchecked", "deprecation"}) // The systemNameList assignment is List<E extends Namedbean> to List<NamedBean>
-                                    // Make this class generic on <E extends NamedBean> (and manager) to fix this.
-                                    // deprecation needs careful unwinding for Set operations
-    public void buildList(jmri.Manager manager) {
+    public void buildList(jmri.Manager<?> manager) {
```

Comment:
Should that be `Manager<?>` or `Manager<E extends Namedbean>`?  (Honest question, don't know if there's a problem with the later form)

Reply:
I was trying to avoid the cast to get to the .getSystemName() method.

NetBeans really didn't like it Manager<E extends Namedbean> , 
but seems happy enough with Manager<?> and finds the expected manager classes fine :-)